### PR TITLE
Updates registry_admin version number + CHANGELOG

### PR DESCRIPTION
**Pull request by oliviawongnyc**
**Change**
- Updated registry_admin version number from v1.4.14 to v1.4.15.

**Reason**
- Updated the SearchForm component in the registry_admin to display a loading message before the search is completed. Need to update the version in the library_registry to deploy this new feature to QA.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "author": "NYPL",
   "license": "Apache-2.0",
   "dependencies": {
-    "simplified-registry-admin": "1.4.14"
```
Comment (EdwinGuzman) [inline package.json:13]: Update the version above as well.

Comment (oliviawongnyc) [reply package.json:13]: I think it is updated? See below!

Comment (EdwinGuzman) [reply package.json:13]: The registry admin package was updated but this app should be updated as well. In the changelog you have the new version as 1.0.11 so it should be updated on line 3 as well.

Comment (oliviawongnyc) [reply package.json:13]: Oh shoot!

**Review by EdwinGuzman — APPROVED** [review]

**Pull request merged**